### PR TITLE
adding policies page to the docs

### LINKED ISSUE #3056: Policies missing from docs
Cross posting from https://github.com/rust-lang/crates.io/issues/408 because the change affects this repo.

The crates.io package policies are not listed on docs.cargo.io.

Add it.

### PR DESCRIPTION
**Pull request by rjgoldsborough**
Fixes #3056

### CODE REVIEW

**Comment by steveklabnik** [comment]
Thanks @rjgoldsborough . Two quick questions:

1. I'm assuming that `metadata` stuff was added because this is the first time it's been run on a cargo that supports it? @alexcrichton ?

2. I think I'd like to make some fixes, but I might send a PR to your PR for them. Thoughts on that? You'd merge my PR to your PR, then we'd merge this PR, which would then include the fixes :smile:

**Comment by rjgoldsborough** [comment]
@steveklabnik,

Yeah, I just straight copied all that text so it might need some updating.

But sure. Whatever works for you. Just excited to dive in. :)

And honestly, I'm not sure why that got added heh. It came with the build so I left it.

**Comment by steveklabnik** [comment]
Sent!

**Comment by steveklabnik** [comment]
Also, /cc @rust-lang/core , given that this is no longer the literal text of the old post. I've mostly

1. Removed speculation about future features
2. Reorganized and added headings
3. Removed other wording that was better suited for the previous announcement, rather than a more static policies page.

File: src/doc/policies.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+existing maintainer can add them as an owner, and the new maintainer can remove
+them. If necessary, the team may reach out to inactive maintainers and help
+mediate the process of ownership transfer.
+
+# Removal
+
+Many questions are specialized instances of a more general form: “Under what
+circumstances can a package be removed from Crates.io?”
+
+The short version is that packages are first-come, first-served, and we won’t
```
Comment (alexcrichton) [inline src/doc/policies.md]: "served"

**Comment by alexcrichton** [comment]
@steveklabnik looks good to me!

**Comment by aturon** [comment]
LGTM

**Comment by steveklabnik** [comment]
@bors: r+ rollup

thanks! Sorry for the latency here, it's been a busy month!

**Comment by rjgoldsborough** [comment]
Yeah for sure, and no problem!

**Comment by rjgoldsborough** [comment]
And thank y'all!

**Pull request merged**